build.gradle

```diff
@@ -25,4 +25,6 @@ dependencies {
   compile 'org.jsoup:jsoup:1.11.3'
 
   compile 'net.iharder:base64:2.3.9'
+
+  compile 'com.google.guava:guava:23.5-jre'
```

Comment:
Pulling in this huge messy dependency just for the cache is overkill. Using this lib is a better idea: https://github.com/ben-manes/caffeine

Reply:
Let me change that, but I'll also close this PR, cause it doesn't really make sense to PR a fork of DevYukine's LP into Fred's LP.